internal/gatewayapi/testdata/tlsroute-with-backend.out.yaml

```diff
@@ -121,8 +121,6 @@ xdsIR:
       routes:
       - destination:
           name: tlsroute/default/tlsroute-1/rule/-1
```

Comment:
should the destination exist in this case?

Reply:
nope, updated.